### PR TITLE
Fixed a few links which pointed to azure.microsoft.com/documentation

### LINKED ISSUE #3452: Fix Azure links
Links that go to either https://azure.microsoft.com/documentation/* or https://azure.microsoft.com/en-us/documentation/* now have new URLs. We should update those links.

A couple of notes:
- For links inside docs.microsoft.com, we use relative paths except for the following e-book folders:
  - https://github.com/dotnet/docs/tree/master/docs/standard/containerized-lifecycle-architecture
  - https://github.com/dotnet/docs/tree/master/docs/standard/microservices-architecture
- When using the absolute URL, we remove the locale such as en-us, to be locale agnostic.
For example, from https://azure.microsoft.com/documentation/articles/vs-azure-tools-docker-edit-and-refresh/ to https://docs.microsoft.com/azure/vs-azure-tools-docker-edit-and-refresh
- For relative paths to docs.microsoft.com that are not under dotnet, the URL https://docs.microsoft.com/azure/vs-azure-tools-docker-edit-and-refresh would have the link URL as /azure/vs-azure-tools-docker-edit-and-refresh

### PR DESCRIPTION
**Pull request by sputier**
Fixes #3452 

There were few links which needed to be changed, mainly under the containerized-lifecycle-architecture e-book folder. 4 links hadn't a counterpart under http://docs.microsoft.com : 
 
- https://azure.microsoft.com/documentation/samples/storage-blob-dotnet-getting-started/
- https://azure.microsoft.com/documentation/templates/
- https://azure.microsoft.com/documentation/samples/storage-table-dotnet-getting-started/
- https://azure.microsoft.com/en-us/documentation/articles/websites-webjobs-resources/ => links to GitHub

@mairaw for review

### CODE REVIEW

File: docs/standard/containerized-lifecycle-architecture/design-develop-containerized-apps/orchestrate-high-scalability-availability.md [hunk 1]
```diff
@@ -120,19 +120,19 @@ Figure 4-9: Deploying microservices as processes or as containers in Azure Servi
...
+As shown in Figure 4-10, and thinking from a logical/business microservice perspective, when implementing a Service Fabric Stateful Reliable Service, you usually will need to implement two tiers of services. The first is the back-end stateful reliable service, which handles multiple partitions. The second is the front-end service, or Gateway service, in charge of routing and data aggregation across multiple partitions or stateful service instances. That Gateway service also handles client-side communication with retry loops accessing the back-end service used in conjunction with the Service Fabric [reverse proxy](https://docs.microsoft.com//azure/service-fabric/service-fabric-reverseproxy).
```
Comment (mairaw) [inline docs/standard/containerized-lifecycle-architecture/design-develop-containerized-apps/orchestrate-high-scalability-availability.md]: double slashes on the last link

File: docs/standard/containerized-lifecycle-architecture/design-develop-containerized-apps/visual-studio-tools-for-docker.md [hunk 2]
```diff
@@ -62,10 +62,10 @@ You could deploy a multicontainer application by using a single docker-compose.y
 **More info:** For further details on the services implementation and use of Visual Studio Tools for Docker, read the following articles:
 
 Build, debug, update, and refresh apps in a local Docker container: [https://azure.microsoft.com/\
-documentation/articles/vs-azure-tools-docker-edit-and-refresh/](https://azure.microsoft.com/documentation/articles/vs-azure-tools-docker-edit-and-refresh/)
```
Comment (mairaw) [inline docs/standard/containerized-lifecycle-architecture/design-develop-containerized-apps/visual-studio-tools-for-docker.md]: should update the link text too

File: docs/standard/containerized-lifecycle-architecture/design-develop-containerized-apps/visual-studio-tools-for-docker.md [hunk 3]
```diff
@@ -62,10 +62,10 @@ You could deploy a multicontainer application by using a single docker-compose.y
...
+documentation/articles/vs-azure-tools-docker-edit-and-refresh/](https://docs.microsoft.com/azure/vs-azure-tools-docker-edit-and-refresh)
 
 Deploy an ASP&period;NET container to a remote Docker host: [https://azure.microsoft.com/\
-documentation/articles/vs-azure-tools-docker-hosting-web-apps-in-docker/](https://azure.microsoft.com/documentation/articles/vs-azure-tools-docker-hosting-web-apps-in-docker/)
```
Comment (mairaw) [inline docs/standard/containerized-lifecycle-architecture/design-develop-containerized-apps/visual-studio-tools-for-docker.md]: should update link text too

**Review by mairaw — CHANGES_REQUESTED** [review]
This is great @sputier! I wanted to get those links updated. Left just a few comments to be addressed,

**Comment by sputier** [comment]
Thanks for the review @mairaw . I fixed these issues !

**Review by mairaw — APPROVED** [review]
Thanks for making the fixes so promptly! I'll merge as soon as the build finishes successfully!

**Comment by mairaw** [comment]
Build looks good. Your changes should show up live in the next day or so.

**Pull request merged**